features/exploredash/src/main/java/org/dash/wallet/features/exploredash/repository/ExploreRepository.kt

```diff
@@ -197,7 +197,7 @@ class GCExploreDatabase @Inject constructor(
         configScope.launch {
             val prefs = exploreConfig.exploreDatabasePrefs.first()
             exploreConfig.saveExploreDatabasePrefs(
-                prefs.copy(localDbTimestamp = max(prefs.localDbTimestamp, updateTimestampCache))
+                prefs.copy(localDbTimestamp = updateTimestampCache)
```

Comment:
the updateTimestampCache value could be older than the local db timestamp if forcing a preload